src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+@import '../styles/variables';
+@import '../styles/utils/mixins';
+
+.card {
+  width: $card-width;
+
+  @include flex-center;
+
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: calc($card-width - 40px);
+    height: 134px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: $font-size-base;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 $small-margin;
+    color: $main-accent-color;
+
+    .card:hover & {
```

Comment:
The selector '.card:hover &' is not necessary here. Since you are already inside the '.card__title' block, you can use '&:hover' to achieve the same effect.